README.md

```diff
@@ -83,14 +83,13 @@ Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 
 (open a pull request if you want your project to be added here)
 
-- https://github.com/paritytech/polkadot
-- https://github.com/paritytech/substrate
-- https://github.com/sigp/lighthouse
-- https://github.com/golemfactory/golem-libp2p
-- https://github.com/comit-network
-- https://github.com/rs-ipfs/rust-ipfs
-- https://github.com/marcopoloprotocol/marcopolo
-- https://github.com/ChainSafe/forest
-- https://github.com/ipfs-rust/ipfs-embed
-- https://www.actyx.com/developers/
-- https://github.com/starcoinorg/starcoin
+- [COMIT](https://github.com/comit-network/xmr-btc-swap) - Bitcoin–Monero Cross-chain Atomic Swap.
+- [Forest](https://github.com/ChainSafe/forest) - An implementation of Filecoin written in Rust.
+- [Lighthouse](https://github.com/sigp/lighthouse) - Ethereum consensus client in Rust.
+- [Locutus](https://github.com/freenet/locutus) - Global, observable, decentralized key-value store.
+- [Starcoin](https://github.com/starcoinorg/starcoin) - A smart contract blockchain network that scales by layering.
+- [Subspace](https://github.com/subspace/subspace) - Subspace Network reference implementation
+- [Substrate](https://github.com/paritytech/substrate) - Framework for blockchain innovation.
```

Comment:
Suggestion to reference Polkadot here makes sense to me.

Reply:
thanks Max, updated :)